### DISCUSSION TITLE
Problema em conectar dispositivos

### DISCUSSION THREAD

**Discussion by kakareco** [blueman-project/blueman#2172] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

<!--

ℹ Please use some service like [Pastebin](http://pastebin.com/) or [GitHub Gist](https://gist.github.com/) to post your logs and keep the thread clean and readable. Make sure to describe the exact steps you took when producing it.

->Falha de Conexão: br-connection-profile-unavailable

**Comment by infirit** [comment]
https://github.com/blueman-project/blueman/wiki/Troubleshooting
